### ISSUE TITLE
should only display two dots.

### ISSUE THREAD

**Issue by misak1** [kenwheeler/slick#2158]
https://jsfiddle.net/misak1/qn0ur4ps/2/

====================================================================

#### Steps to reproduce the problem

https://jsfiddle.net/misak1/qn0ur4ps/2/

```
$(".slider").slick({
    autoplay: true,
    dots: true,
    arrows: true,
    infinite: false,
    slidesToShow: 4,
    slidesToScroll: 1
});
```

====================================================================

#### What is the expected behaviour?

should only display two dots.

====================================================================

#### What is observed behaviour?

display 5 dots.

====================================================================

**Comment by mazzaker** [comment]
Duplicate to #2146 

Fixed with #2152 

[Fiddle](https://jsfiddle.net/jL1j92x5/1/)

**Comment by leggomuhgreggo** [comment]
@mazzaker Thanks man!

Closing
